### PR TITLE
Update to UCOW

### PR DESCRIPTION
**Pull request by ddbb07**
- Added "after:" messages for UltimateCollege.esp
- Added "inc:" messages for UltimateCollege.esp

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -19150,11 +19150,19 @@ plugins:
...
+      - 'mihailspectres.esp'
+      - 'mihailbeholder.esp'
+      - 'ImmersiveIntegrationSE15.esp'
     inc:
       - 'CollegeModifications.esp'
       - 'MagicalCollegeofWinterhold.esp'
       - 'OCW_LightingPatch_Brighter.esp'
       - 'OCW_LightingPatch_Darker.esp'
+      - name: 'mihailspectres.esp'
+        condition: 'active("mihailspectres.esp") and checksum("mihailspectres.esp", 0x95F0F68D)'
```
Comment (sibir-ine) [inline masterlist.yaml]:
```suggestion
        condition: 'active("mihailspectres.esp") and checksum("mihailspectres.esp", 95F0F68D)'
```

File: masterlist.yaml [hunk 2]
```diff
@@ -19150,11 +19150,19 @@ plugins:
...
+      - name: 'mihailbeholder.esp'
+        condition: 'active("mihailbeholder.esp") and checksum("mihailbeholder.esp", 0x447993FF)'
```
Comment (sibir-ine) [inline masterlist.yaml]:
```suggestion
        condition: 'active("mihailbeholder.esp") and checksum("mihailbeholder.esp", 447993FF)'
```

**Review by sibir-ine — CHANGES_REQUESTED** [review]
@ddbb07

File: masterlist.yaml (hunk 1 above)
Comment (MacSplody) [inline masterlist.yaml]: You don't need the prefix of the CRC when using `checksum()`.
```suggestion
        condition: 'active("mihailspectres.esp") and checksum("mihailspectres.esp", 95F0F68D)'
```

Comment (ddbb07) [reply masterlist.yaml]: Sorry for that

**Review by MacSplody — APPROVED** [review]
Looks fine to me, @sibir-ine Are you happy with it?

**Review by sibir-ine — APPROVED** [review]

**Comment by sibir-ine** [comment]
Yeah, this is what @themayor897 was talking about in Discord.

**Comment by MacSplody** [comment]
@ddbb07 Thanks.

**Pull request merged**